index.bs

```diff
@@ -203,6 +203,9 @@ Styles applied to |video| (such as opacity, visibility, transform, etc.) MUST
 NOT apply in the Picture-in-Picture window. Its aspect ratio is based on the
 video size.
 
+It is also RECOMMENDED that the Picture-in-Picture window has a maximum size
+set to half of the screen.
```

Comment:
I think we should say maximum and minimum and give examples. Something like:
"It is also RECOMMENDED that the Picture-in-Picture window has a maximum and minimum size. For example, it could be restricted to be between a quarter and a half of one dimension of the screen."

Reply:
Done.